randovania/games/samus_returns/logic_database/Area 2 Dam Exterior.txt

```diff
@@ -178,23 +174,37 @@ Extra - asset_id: collision_camera_005
   * Layers: default
   > Elevator to Area 2 Dam Entryway
       Trivial
-  > Door to Arachnus Arena
-      Trivial
   > Door to Rock Icicle Corridor
       Trivial
   > Door to Exterior Alpha+ Arena
       Diagonal Bomb Jump (Advanced) and Lay Bomb
+  > Next to Arachnus Arena
+      Trivial
 
 > Start Point (Top); Heals? False; Spawn Point
   * Layers: default
   * Extra - start_point_actor_name: ST_SG_Alpha_001
   > Door to Exterior Alpha+ Arena
       Trivial
 
-> Start Point (Bottom); Heals? False; Spawn Point
+> Next to Arachnus Arena; Heals? False; Spawn Point
   * Layers: default
   * Extra - start_point_actor_name: ST_SG_Arachnus
   > Door to Arachnus Arena
+      Any of the following:
+          Disabled Final Boss Arachnus
+          Enabled Final Boss Arachnus and All DNA
+  > Door from Maintenance Tunnel
+      Trivial
+  > Top of Wall
+      Single-wall Wall Jump (Intermediate) or Climb Rooms Vertically (No High Jump)
+  > Event - Credits
```

Comment:
when exactly does the game go to credits when beating arachnus? immediately after arachnus is dead? asking cause this seems like it would cause a mismatch of requirements.

Reply:
When you re-enter Dam Exterior. I guess I only mentioned it patcher side and not here.

Reply:
Gothca.